beeswax/openrtb/extension.proto

```diff
@@ -303,6 +305,10 @@ message UserExtensions {
 
   // Number of days since the user cookie has been dropped, for example, 10
   optional int32 cookie_age = 3;
+  // map from dynamic bid modifier token to multiplier.
+  // Since dynamic modifiers are applied on user/list level,
+  // the key should always be a token rather than a boolean expression.
+  map<string, float> dynamic_modifiers_map = 5;
```

Comment:
I don't think we want to make this change in public repo until bid modifier project is done